### PR TITLE
Add Tim to about page

### PR DESCRIPTION
**Pull request by timhaines**

### CODE REVIEW

**Comment by timhaines** [comment]
image

File: app/templates/about.hbs [hunk 1]
```diff
@@ -27,20 +27,17 @@
...
     </div>
     <div class="nine columns">
       <h4>Mike Fotinakis</h4>
-      <h5>Founder, CEO</h5>
+      <h5>Co-Founder, CEO</h5>
       <p>
         I've been breaking computers since I was 10 and learning how to fix them along the way. I've worked at Google as a software engineer on the global web team, helping build the distributed tools and platforms underlying all of Google's websites, and at <a href="https://www.scienceexchange.com/">Science Exchange</a>, learning the ins and outs of startup engineering. I founded Percy to help solve the hard problems of visual QA for as many engineering teams as possible.
       </p>
       <p>
-        A big thank you to my talented advisors and friends for their continued support and thoughtful advice.
```
Comment (timhaines) [inline app/templates/about.hbs]: Hey @fotinakis - I wanted to call this change out in particular.  It looked a little out of place when I first started adding myself, and it sort of calls out other "contributors" who are invisible.  Now we have 2 on the page, it seems to detract?   But I'm totally cool with adding it back in.  😄

**Comment by timhaines** [comment]
image

File: app/styles/_marketing.sass [hunk 2]
```diff
@@ -82,6 +82,9 @@
     a
       margin-bottom: 20px
 
+  .row.bio
```
Comment (fotinakis) [inline app/styles/_marketing.sass]: Change `.bio` to `.AboutContainer-header` to match our [SUIT](https://suitcss.github.io/) CSS naming conventions.

File: app/templates/about.hbs (hunk 1 above)
Comment (fotinakis) [inline app/templates/about.hbs]: Sounds good to me.

**Pull request merged**